test/unit/rules/lint-no-abstract-roles-test.js

```diff
@@ -0,0 +1,149 @@
+'use strict';
+
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+generateRuleTests({
+  name: 'no-abstract-roles',
+
+  config: true,
+
+  good: [
+    '<img alt="" role="none" src="zoey.jpg">',
+    '<img alt="" role="presentation" src="zoey.jpg">',
+  ],
+
+  bad: [
+    {
+      template: '<img role="command">',
+
+      result: {
+        message: 'The HTML attribute role should never have the following value: command',
+        moduleId: 'layout.hbs',
+        source: '<img role="command">',
+        line: 1,
+        column: 0,
+      },
+    },
+    {
+      template: '<img role="composite">',
+
+      result: {
+        message: 'The HTML attribute role should never have the following value: composite',
```

Comment:
please refer to the above change request and update the others to match. thank you!